docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Listing all persons sorted by their names : `sort`
+
+Shows a list of all persons sorted by their names, along with their non-private details, in the address book. +
+Format: `list`
```

Comment:
I think perhaps the format is wrong? For a sort command, the format should be: 'sort'?

Reply:
Yes thanks for pointing that out!